src/transformers/modeling_flaubert.py

```diff
@@ -287,7 +292,7 @@ def forward(
         outputs = (tensor,)
         if self.output_hidden_states:
```

Comment:
is this still on `config`? Is that desired behavior?

Reply:
yeah the 'output_hidden_states` is going to be replaced in a separate PR: https://github.com/huggingface/transformers/pull/4548 .

It's prob going to be quite a mess with merge conflicts so we will merge this one first and then I rebase the other one